src/grossToNet.js

```diff
@@ -1,10 +1,10 @@
-const Big = require('big.js');
+import Big from 'big.js';
 const todayDate = new Date();
-const calculateTaxes = require('./taxCalculator.js');
-const calcInsuranceDetails = require('./insurancesCalculator.js');
-const {SELF_DEDUCTION, DEDUCTION_PER_PERSON} = require("./salaryConstants");
+import {calculateTaxes} from './taxCalculator';
```

Comment:
Old `const calculateTaxes = require('./taxCalculator.js')`
The new one should be equivalent, can use export default in  `taxCalculator.js`

Reply:
Can check the rest